nimble/host/src/ble_hs_id.c

```diff
@@ -77,8 +87,6 @@ ble_hs_id_set_rnd(const uint8_t *rnd_addr)
 
     memcpy(ble_hs_id_rnd, rnd_addr, 6);
 
-    rc = 0;
```

Comment:
this will left rc uninitialized in case of no error

Reply:
In case of no error, rc will be set in [rc = ble_hs_hci_util_set_random_addr(rnd_addr);](https://github.com/apache/mynewt-nimble/pull/765/files#diff-1544015ed01ed7d5bbb3804be5fb1ba6R83)